### PR TITLE
Use SHARED_DIR as tmp

### PR DESCRIPTION
**Pull request by ngaranko**
Make /tmp directory configurable via env variables.

### CODE REVIEW

File: src/dags/dag_update.py [hunk 1]
```diff
@@ -26,18 +26,18 @@ def doit():
 
     fetch_dags_from_github = BashOperator(
         task_id="fetch_dags_from_github",
-        bash_command=f"wget {github_url} -O /tmp/repo.zip",
+        bash_command=f"wget {github_url} -O {SHARED_DIR}/repo.zip",
```
Comment (gkoller) [inline src/dags/dag_update.py:29]: I think this should be `${SHARED_DIR}`. Eg with a `$`

Comment (ngaranko) [reply src/dags/dag_update.py:29]: it's an f string, not a bash command yet ;)

Comment (gkoller) [reply src/dags/dag_update.py:29]: Oh, no, not here, but you have also `BashOperators` where the string is not a format string. Than it boils down to shell variable expansion and then it does need the `$`

File: src/dags/dag_update.py [hunk 2]
```diff
@@ -26,18 +26,18 @@ def doit():
...
     )
 
     extract_zip = BashOperator(
-        task_id="extract_zip", bash_command="unzip -o /tmp/repo.zip -d /tmp"
+        task_id="extract_zip", bash_command="unzip -o {SHARED_DIR}/repo.zip -d {SHARED_DIR}"
```
Comment (gkoller) [inline src/dags/dag_update.py:33]: Here, is an example. No format string, hence shell variable expansion (I presume), hence the need for a `$`.

**Review by jjmurre — APPROVED** [review]
LGTM.  Er lopen nog een paar DAGs via de vsd route. Die doet z'n eigen configuratie van TMPDIR. Die zou je ook nog op SHARED_DIR kunnen baseren. Of wellicht houd je daar al rekening mee? Daar is ook al een SHARED_DIR variabele.

File: src/vsd/shared/utils/get_objectstore_file.py [hunk 3]
```diff
@@ -36,5 +36,5 @@ def get_objectstore_file(location, dir1, connection):
     parser.add_argument("filename")
     parser.add_argument("-s", "--swift_connection")
     args = parser.parse_args()
-    tmpdir = os.getenv("TMPDIR", "/tmp/")
+    tmpdir = os.getenv("SHARED_DIR", "/tmp/")
```
Comment (gkoller) [inline src/vsd/shared/utils/get_objectstore_file.py:39]: Not in the scope of this PR, but I was wondering how we deal with environment variables. Here we use the `os` module. Most other places use `environs.Env`. We should probably use one way.

With type annotations in mind I'd like to suggest not using `os` or even `environs` but using Pydantic's settings management instead: https://pydantic-docs.helpmanual.io/usage/settings/

File: src/dags/dag_update.py (hunk 2 above)
Comment (ngaranko) [reply src/dags/dag_update.py:33]: Good one, thanks.

File: src/vsd/shared/utils/get_objectstore_file.py (hunk 3 above)
Comment (ngaranko) [reply src/vsd/shared/utils/get_objectstore_file.py:39]: This one is worth discussion with everyone.

**Comment by ngaranko** [comment]
> LGTM. Er lopen nog een paar DAGs via de vsd route. Die doet z'n eigen configuratie van TMPDIR. Die zou je ook nog op SHARED_DIR kunnen baseren. Of wellicht houd je daar al rekening mee? Daar is ook al een SHARED_DIR variabele.

@jjmurre i checked, most of VSD DAGs will be exedcuted as single step and do not really need shared folder. But maybe it's good to change those as well.

File: src/vsd/shared/utils/get_objectstore_file.py (hunk 3 above)
Comment (gkoller) [reply src/vsd/shared/utils/get_objectstore_file.py:39]: Agreed. Already posted something on Slack

**Review by gkoller — APPROVED** [review]

**Pull request merged**